plugins/coral-plugin-offtopic/client/constants.js

```diff
@@ -0,0 +1 @@
+export const TOGGLE_CHECKBOX = 'TOGGLE_CHECKBOX';
```

Comment:
These constants need a global prefix so it doesn't collide with other plugins and the core. This should be something like `CORAL_OFFTOPIC_TOOGLE_CHECKBOX`.